components/CreatePost.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+
+type NewPost = {
+  titleDescription: string;
+  type: string;
+};
+
+const iniState = {
+  titleDescription: "",
+  type: "Life",
+};
+
+export const CreatePost = () => {
```

Comment:
lets use function declaration (named function) instead of arrow functions for standard
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function